examples/docker-compose-proxy/verify.sh

```
@@ -0,0 +1,129 @@
+  fail "docker daemon not running (start Docker Desktop / Engine)"
+  printf '\n\033[1m=== Results: %s passed, %s failed ===\033[0m\n\n' "$PASS" "$FAIL"
+  exit 1
+fi
+
+# -- Test 1: build and start ---------------------------------------------------
+step "Test 1: build and start docker compose"
+(
+  cd "$EXAMPLE_DIR"
+  docker compose build >/dev/null
+  docker compose up -d >/dev/null
+) && pass "compose started" || {
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)